icaruscode/Decode/ChannelMapping/ICARUSChannelMapProvider.cxx

```diff
@@ -63,13 +102,15 @@ ICARUSChannelMapProvider::ICARUSChannelMapProvider(const fhicl::ParameterSet& ps
 
     theClockReadoutIDs.start();
 
+    fReadoutBoardToChannelMap.clear();
     if (fChannelMappingTool->BuildTPCReadoutBoardToChannelMap(fReadoutBoardToChannelMap))
     {
         std::cout << "******* FAILED TO CONFIGURE CHANNEL MAP ********" << std::endl;
         throw cet::exception("ICARUSChannelMapProvider") << "POS didn't read the F'ing database again \n";
     }
 
     // Do the channel mapping initialization
+    fFragmentToDigitizerMap.clear();
     if (fChannelMappingTool->BuildFragmentToDigitizerChannelMap(fFragmentToDigitizerMap))
```

Comment:
Is it possible to rename this function to `BuildPMTFragmentToDigitizerChannelMap`? I think it would be clearer and also in line with the same functions for the other subsystems.

Reply:
That goes against the principle of "not making this code better".